packages/validator/specs/configSecurity.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
Shouldn't this directory be called `templates` and not `specs`?

Reply:
Yes, I'm having some issues with the naming, in the validator `template` is called `specsStruct` (validator specification structure), in one of the PRs Andre suggested to call it template instead, however at that point the validator was already pretty much finished and the specsStruct was used everywhere. So in some more recent commits template is used, but specsStruct is still used in most of places and im not sure if its worth it to go through all the code and rename it everywhere (code, JSDoc, readme etc...) or leave it as it is and rename it only in instances such as this one, what do you think?

Reply:
I personally do this kind of stuff all the time (a commit today for example: https://github.com/api3dao/api3-docs/commit/e55c526b9b4d8d00fdddcf63b04fb10c10e9e6c1) so I would say it's worth it, especially considering that this code will be used by a lot of people for a long time. It doesn't look that bad either, a search returned me 47 results.

Reply:
Renamed in 526ceae